### PR TITLE
Add helper tip string for archive uploads

### PR DESCRIPTION
**Pull request by pgrace-google**
This message is shown in the quick uploads option, but some shepherds use the regular upload, where one has to know to hover to see this information.

crbug/1489702

### CODE REVIEW

**Review by jonathanmetzman — APPROVED** [review]
LGTM

File: src/appengine/private/components/upload-testcase/upload-form.html [hunk 1]
```diff
@@ -276,6 +279,13 @@
             window.query.get(), this.paramKeys);
       }
 
+      shouldShowFiletypePicker(filePath) {
+        if (!filePath) {
+          return false;
+        }
+        return !filePath.includes('.html') && !filePath.includes('.js') && !filePath.includes('.pdf');
```
Comment (jonathanmetzman) [inline src/appengine/private/components/upload-testcase/upload-form.html:286]: Sorry why is this part needed?

Comment (pgrace-google) [reply src/appengine/private/components/upload-testcase/upload-form.html:286]: to show the message only when the file type is an archive! felt like it didnt need to be shown otherwise

**Review by jonathanmetzman — APPROVED** [review]
LGTM
